packages/squiggle-lang/__tests__/TS/KDE.ts

```diff
@@ -0,0 +1,59 @@
+import * as fc from "fast-check";
+import { samplesToContinuousPdf } from "../../src/rescript/Distributions/SampleSetDist/KdeLibrary.js";
+import range from "lodash/range";
+
+const evalKde = (samples: number[], width: number, weight: number, x: number) =>
+  (weight / width) *
+  samples
+    .map((p) => Math.max(0, 1 - Math.abs((p - x) / width)))
+    .reduce((a, b) => a + b);
```

Comment:
I'd probably use Lodash Sum here. (Especially as this doesn't seem performance critical) https://lodash.com/docs/4.17.15#sum